src/main/java/chess/dto/PointsDTO.java

```diff
@@ -0,0 +1,19 @@
+package chess.dto;
+
+public final class PointsDTO {
+    private final String start_point;
```

Comment:
```suggestion
    private final String startPoint;
```

자바에서는 카멜케이스를 사용하는 것이 관례입니다.

Reply:
😢